index.html

```diff
@@ -919,11 +945,100 @@ <h4>Verify Proof (eddsa-jcs-2022)</h4>
             </dd>
           </dl>
 
-          <p>
-The algorithm is the same as the one described in Section
-[[[#verify-proof-eddsa-rdfc-2022]]] with the changes outlined in
-Section [[[#eddsa-jcs-2022]]].
-          </p>
+          <ol class="algorithm">
+            <li>
+Let |unsecuredDocument| be a copy of |securedDocument| with
+the `proof` value removed.
+            </li>
+            <li>
+Let |proofOptions| be the result of a copy of |securedDocument|.|proof| with `proofValue`
+removed.
+            </li>
+            <li>
+Let |proofBytes| be the
+<a data-cite="VC-DATA-INTEGRITY#multibase-0">Multibase decoded base58-btc
+value</a> in |securedDocument|.|proof|.|proofValue|.
+            </li>
+            <li>
+Perform the following checks  between the |securedDocument|.<var>@context</var> and
+the |proofOptions|.<var>@context</var>:
+              <ol class="algorithm">
+                <li>
+The |securedDocument|.<var>@context</var> MUST contain all values contained in the
+|proofOptions|.<var>@context</var> in the same order. Otherwise, set |verified| to
+`false` and skip to the last step.
+                </li>
+                <li>
+The |securedDocument|.<var>@context</var> MUST NOT have any values preceding the
+values contained in the |proofOptions|.<var>@context</var>, but those MAY be followed by
+additional values not contained in the |proof|.<var>@context</var>. Otherwise,
+set |verified| to `false` and skip to the last step.
+                </li>
+              </ol>
+
+            </li>
+            <li>
+Set |securedDocument|.<var>@context</var> equal to
+|proofOptions|.<var>@context</var>.
```

Comment:
I think we can move this "Set..." step into the "If" conditional I suggested above for backwards compatibility.